### PR TITLE
[OSDEV-802] API. Track errors that come from API users vs website.

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Fix [OSDEV-802](https://opensupplyhub.atlassian.net/browse/OSDEV-802)

1. Update Rollbar notifications in React app
2. Update Rollbar notifications in Django app
3. Update Rollbar notifications in Dedupe Hub
4. Write tests for ErrorBoundary component

### CODE REVIEW

File: src/dedupe-hub/api/app/utils/rollbar.py [hunk 1]
```diff
@@ -9,13 +9,16 @@
 def init_rollbar():
     if not DEBUG:
         rollbar.init(**ROLLBAR)
-
+
```
Comment (killev) [inline src/dedupe-hub/api/app/utils/rollbar.py:12]: this seems a formatting issue

**Review by killev — CHANGES_REQUESTED** [review]
@VadimKovalenkoSNF please split this PR into the following:
* front-end part
* dedupe-hub part
* django part
and merge them separately.

File: src/dedupe-hub/api/app/utils/rollbar.py (hunk 1 above)
Comment (VadimKovalenkoSNF) [reply src/dedupe-hub/api/app/utils/rollbar.py:12]: My editor always adds a new line at the end of the files. Shall I remove this rule for Python?

Comment (roman-stolar) [reply src/dedupe-hub/api/app/utils/rollbar.py:12]: As I remember end file line is correct behavior.

Comment (VadimKovalenkoSNF) [reply src/dedupe-hub/api/app/utils/rollbar.py:12]: Will check, thanks

**Comment by VadimKovalenkoSNF** [comment]
Abandoned in favor of smaller PRs:

1. Update Rollbar notifications in React app https://github.com/opensupplyhub/open-supply-hub/pull/68
2. Update Rollbar notifications in Django app https://github.com/opensupplyhub/open-supply-hub/pull/69
3. Update Rollbar notifications in Dedupe hub app https://github.com/opensupplyhub/open-supply-hub/pull/74

**Pull request closed without merging**